app/src/main/java/org/mozilla/fenix/components/accounts/FenixAccountManager.kt

```diff
@@ -5,29 +5,55 @@
 package org.mozilla.fenix.components.accounts
 
 import android.content.Context
-import mozilla.components.service.fxa.manager.FxaAccountManager
 import org.mozilla.fenix.ext.components
 
 /**
- * Component which holds a reference to [FxaAccountManager]. Manages account authentication,
- * profiles, and profile state observers.
+ * Contains helper methods for querying FXA account state and it's properties.
```

Comment:
```suggestion
 * Contains helper methods for querying the FxA account state and its properties.
```

Instead of FXA, we should use FxA which is its proper acronym. Alternatively, I would just spell out "Firefox Account" for outside readers who aren't aware of our acronym for internal systems.

Reply:
Thanks, will use "Firefox Account".